test/api/cpp/finite_field.cpp

```diff
@@ -0,0 +1,63 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Alex Ozdemir
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2022 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * An example of solving finite field problems with cvc5's cpp API
+ */
+
+#include <cassert>
+#include <iostream>
+
+#include "api/cpp/cvc5.h"
+
+using namespace std;
+using namespace cvc5;
+
+int main()
```

Comment:
Isn't this the same one as in the examples? What purpose does this serve in addition?

Reply:
Good catch.